airbyte-integrations/connectors/source-the-guardian-api/unit_tests/conftest.py

```diff
@@ -0,0 +1,32 @@
+# Copyright (c) 2024 Airbyte, Inc., all rights reserved.
```

Comment:
TODO: add comments clarifying that this is boilerplate code, don't modify it, blah blah